geopandas/tools/crs.py

```diff
@@ -16,21 +14,12 @@ def explicit_crs_from_epsg(crs=None, epsg=None):
     epsg : string or int, default None
        The EPSG code to lookup
     """
-    if epsg is None and crs is not None:
-        epsg = epsg_from_crs(crs)
-    if epsg is None:
-        raise ValueError(
-            "No epsg code provided or epsg code could not be identified "
-            "from the provided crs."
-        )
-
-    _crs = re.search(r"\n<{}>\s*(.+?)\s*<>".format(epsg), get_epsg_file_contents())
-    if _crs is None:
-        raise ValueError('EPSG code "{}" not found.'.format(epsg))
-    _crs = fiona.crs.from_string(_crs.group(1))
-    # preserve the epsg code for future reference
-    _crs["init"] = "epsg:{}".format(epsg)
-    return _crs
+    warnings.warn(
+        "explicit_crs_from_epsg is deprecated. "
+        "You can set the epsg on the GeoDataFrame (gdf) using gdf.crs=epsg",
+        DeprecationWarning,
```

Comment:
You can make this a FutureWarning I think (it's a bit more visible, I don't think many people rely on this function). Can you also add a `stacklevel=2` ?

(same for below)

Reply:
Sure